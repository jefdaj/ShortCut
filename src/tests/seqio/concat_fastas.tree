.
|-- cache
|   |-- each
|   |   `-- a2ff74da29
|   |       |-- f61974bf8f.fna -> ../../../exprs/concat_fna/f61974bf8f.fna
|   |       `-- f61974bf8f.fna.args -> ../../../cache/lines/6fde87b86e.txt
|   |-- lines
|   |   |-- 0229d38a63.txt
|   |   |-- 5bb3f27328.txt
|   |   |-- 5c59577fb3.txt
|   |   |-- 691dd4d722.txt
|   |   |-- 6fde87b86e.txt
|   |   |-- cad7634989.txt
|   |   `-- eed7b93561.txt
|   `-- load
|       |-- d8da51f7cd.gbk -> $WORKDIR/data/SynPCC7942_pANL.gbk
|       `-- ec9540d17e.gbk -> $WORKDIR/data/SynPCC7942_chr.gbk
|-- exprs
|   |-- any
|   |   `-- debc54b62a.fna.list -> ../../cache/lines/5c59577fb3.txt
|   |-- concat_fna
|   |   |-- eb1aea4e88.fna
|   |   `-- f61974bf8f.fna
|   |-- concat_fna_each
|   |   `-- 72514253ee.fna.list -> ../../cache/lines/0229d38a63.txt
|   |-- gbk_to_fna
|   |   |-- 1a2d376906.fna
|   |   `-- 455f27117a.fna
|   |-- list
|   |   |-- 20812f3e16.fna.list.list -> ../../cache/lines/eed7b93561.txt
|   |   |-- 51e8fbf7e2.fna.list.list -> ../../cache/lines/6fde87b86e.txt
|   |   |-- 53aa247bf9.fna.list -> ../../cache/lines/5bb3f27328.txt
|   |   `-- de88435002.fna.list -> ../../cache/lines/5c59577fb3.txt
|   |-- load_gbk
|   |   |-- 4f798984f2.gbk -> ../../cache/load/d8da51f7cd.gbk
|   |   `-- c5cc3f4b63.gbk -> ../../cache/load/ec9540d17e.gbk
|   `-- str
|       |-- 7ec89da918.str -> ../../cache/lines/cad7634989.txt
|       `-- a8d18fa355.str -> ../../cache/lines/691dd4d722.txt
`-- vars
    |-- mapped.fna.list -> ../exprs/concat_fna_each/72514253ee.fna.list
    |-- result -> ../exprs/any/debc54b62a.fna.list
    `-- single.fna -> ../exprs/concat_fna/eb1aea4e88.fna

14 directories, 28 files
