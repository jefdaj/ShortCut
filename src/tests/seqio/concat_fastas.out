[>Synpcc7942_0001 ABB56033.1 DNA polymerase III, beta subunit non-experimental...
ATGCTTTGGCAAGATTGCGATCAAAGGCTCGGGCAGCCTCCCCCCATGAAGTTGGTCTGTCGCCAAAACGAACTGAA...
>Synpcc7942_0002 ABB56034.1 hypothetical protein non-experimental evidence, n...
GTGGCAATGTCCGACCAGCCTGAAGAACTGCGGGTTAGCGACCTTCTCAATCGACAAGTTCTTGATCGCCAAACCAC...
>Synpcc7942_0003 ABB56035.1 phosphoribosylformylglycinamidine synthase subuni...
...
,
 >Synpcc7942_0001 ABB56033.1 DNA polymerase III, beta subunit non-experimental...
ATGCTTTGGCAAGATTGCGATCAAAGGCTCGGGCAGCCTCCCCCCATGAAGTTGGTCTGTCGCCAAAACGAACTGAA...
>Synpcc7942_0002 ABB56034.1 hypothetical protein non-experimental evidence, n...
GTGGCAATGTCCGACCAGCCTGAAGAACTGCGGGTTAGCGACCTTCTCAATCGACAAGTTCTTGATCGCCAAACCAC...
>Synpcc7942_0003 ABB56035.1 phosphoribosylformylglycinamidine synthase subuni...
...
]
